spec/facter/facts/freebsd/memory/swap/capacity_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+describe Facts::Freebsd::Memory::Swap::Capacity do
+  describe '#call_the_resolver' do
+    it 'returns a fact' do
+      expected_fact = double(Facter::ResolvedFact, name: 'memory.swap.capacity', value: 1024)
```

Comment:
We should use verifying doubles over normal doubles.

```suggestion
      expected_fact = instance_double(Facter::ResolvedFact, name: 'memory.swap.capacity', value: 1024)
```

Reply:
Just noticed, this is the old model for writing unit test for facts. We should use the new model, like you have used in `spec/facter/facts/freebsd/memory/swap/encrypted_spec.rb`